jason/flutter/lib/room_handle.dart

```diff
@@ -275,26 +335,46 @@ class RoomHandle {
     }
   }
 
-  // TODO: Add throws docs when all errros are implemented.
   /// Enables inbound audio in this `Room`.
+  ///
+  /// Throws a [StateError] if the underlying [Pointer] has been freed.
+  ///
+  /// Throws a `MediaStateTransitionException` if [RoomHandle.enableVideo] was
+  /// called while enabling or a media server didn't approve this state
+  /// transition.
   Future<void> enableRemoteAudio() async {
     await (_enableRemoteAudio(ptr.getInnerPtr()) as Future);
   }
 
-  // TODO: Add throws docs when all errros are implemented.
   /// Disables inbound audio in this `Room`.
+  ///
+  /// Throws a [StateError] if the underlying [Pointer] has been freed.
+  ///
+  /// Throws a `MediaStateTransitionException` if [RoomHandle.enableRemoteAudio] was
+  /// called while disabling or a media server didn't approve this state
+  /// transition.
   Future<void> disableRemoteAudio() async {
     await (_disableRemoteAudio(ptr.getInnerPtr()) as Future);
   }
 
-  // TODO: Add throws docs when all errros are implemented.
   /// Enables inbound video in this `Room`.
+  ///
+  /// Throws a [StateError] if the underlying [Pointer] has been freed.
+  ///
+  /// Throws a `MediaStateTransitionException` if [RoomHandle.disableVideo] was
+  /// called while enabling or a media server didn't approve this state
+  /// transition.
   Future<void> enableRemoteVideo() async {
     await (_enableRemoteVideo(ptr.getInnerPtr()) as Future);
   }
 
-  // TODO: Add throws docs when all errros are implemented.
   /// Disables inbound video in this `Room`.
+  ///
+  /// Throws a [StateError] if the underlying [Pointer] has been freed.
+  ///
+  /// Throws a `MediaStateTransitionException` if [RoomHandle.disableVideo] was
+  /// called while disabling or a media server didn't approve this state
+  /// transition.
   Future<void> disableRemoteVideo() async {
```

Comment:
It seems that "Throws a `MediaStateTransitionException`" doc here and in other definitions of this file is not accurate enough. I cannot grasp the pattern, because different function reffer the same method in doc, without feasible logic.

Please, re-read caefully through all the docs in this file once again, and fix methods names for `MediaStateTransitionException`.

Reply:
@tyranron ,

Ошибки в доке поправлю. Логика там должна быть такая:
1. Жмакаем `disableRemoteVideo()`, получаем `Promise1`.
2. Жмакаем `enableRemoteVideo()` (операцию противоположную тому что жмакнули на первом шаге), получаем `Promise2`, а `Promise1` реджектится с `MediaStateTransitionException`.